charts/__tests__/ExploreView.test.tsx

```diff
@@ -24,18 +24,16 @@ const indicatorsJson = fs.readFileSync("test/fixtures/indicators.json")
 const indicatorsUrl = /\/explore\/indicators\.json/
 const indicator: Indicator = JSON.parse(indicatorsJson.toString()).indicators[0]
 
-function getStore() {
-    return new RootStore()
-}
+const store = new RootStore()
```

Comment:
This means the store state changes get shared from one test to another. It feels like we'd want to avoid that?

Reply:
Hmm, good point. I'll take a look, maybe better to create a new store each time

Reply:
Done